### ISSUE TITLE
wikibase-person BIRTH-DATE returns name, not date

### ISSUE THREAD

**Issue by sfelshin** [infolab-csail/WikipediaBase#215]
```
$ telnet kiribati 8023
(get "wikibase-person" "Marvin Minsky" (:CODE "BIRTH-DATE"))
((:html "Marvin Lee Minsky"))
```

DEATH-DATE, on the other hand, works just fine:
```
$ telnet kiribati 8023
(get "wikibase-person" "Marvin Minsky" (:CODE "DEATH-DATE"))
((:yyyymmdd 20160124))
```

**Comment by alvaromorales** [comment]
@chsamlee if this was fixed in #207, please add a test case in #214

**Comment by chsamlee** [comment]
It looks like it is an issue fixed in #209, but I will add a test case for it regardless.
